anvil/src/input_handler.rs

```diff
@@ -172,7 +174,9 @@ impl<Backend> AnvilState<Backend> {
         // subsurface menus (for example firefox-wayland).
         // see here for a discussion about that issue:
         // https://gitlab.freedesktop.org/wayland/wayland/-/issues/294
-        if !pointer.is_grabbed() && !keyboard.is_grabbed() {
+        // Input methods is a special case in that they only use the keyboard input,
```

Comment:
Small grammar nitpick:
```suggestion
        // Input methods are a special case in that they only use the keyboard input,
```
`methods` is plural, therefore `are` is more appropriate
Or perhaps it should be:
```suggestion
        // Input method is a special case in that they only use the keyboard input,
```